operator/Makefile

```diff
@@ -112,11 +112,11 @@ RUN = $(CONTAINER_CLI) run -t -i --sig-proxy=true -u $(UID):$(GID) --rm \
 	-v /etc/passwd:/etc/passwd:ro \
 	$(DOCKER_SOCKET_MOUNT) \
 	$(CONTAINER_OPTIONS) \
-	--mount type=bind,source="$(PWD)",destination="/work" \
+	--mount type=bind,source="$(PWD)/..",destination="/work" \
```

Comment:
These changes are needed to get this makefile to work in the docker container, because it expects `../common-protos` to exist. Can put into a separate PR if desired